phobos_umss.sh

```diff
@@ -0,0 +1,490 @@
+#!/bin/bash
+
+## Copyright (c) 2009 Data Intensive Cyberinfrastructure Foundation. All rights reserved.
+## For full copyright notice please refer to files in the COPYRIGHT directory
+## Written by Jean-Yves Nief of CCIN2P3 and copyright assigned to Data Intensive Cyberinfrastructure Foundation
```

Comment:
question: is it the licence for iRODS components? can we put the one we want?

Reply:
This is the copyright notice from iRODS UnivMSS template.
iRODS is under BSD licence, described in `https://github.com/irods/irods/blob/main/LICENSE`.
A correction to meet the requirements has been commited.

Reply:
The most relevant norm for this script is actually the same as Phobos : LGPL2.1.
Will be modified in the next commit.